<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddExtraMaterialOptionDialog</class>
 <widget class="QWidget" name="AddExtraMaterialOptionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSpinBox" name="valueBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>43</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameLine">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>260</y>
     <width>83</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QComboBox" name="measureBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>210</y>
     <width>79</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>см</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>мм</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>см2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>мм2</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="errorLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>57</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
